### DISCUSSION TITLE
Script: Remove `liq_` tags from media files to fix Autocue issues

### DISCUSSION THREAD

**Discussion by RM-FM** [AzuraCast/AzuraCast#7563] [General]
I created a script designed to remove all `liq_` tags from media files. This can be useful if you previously enabled the "store tags to file" option in Azuracast's older `autocue.cue_in` implementation. These tags are conflict with the updated Azuracast Rolling Release, which uses Liquidsoap 2.3.x's `autocue.internal`.

It also comes with a dry run feature to initially check your files without making any changes.

https://github.com/RM-FM/liq-tag-removal

> [!IMPORTANT]
> Please also ensure that you have cleared the Extra Metadata in Azuracast's Media File Manager, as older versions of Azuracast may have stored these values there as well:
>
>`Media > Music Files > set search to show ALL > select ALL > More > Clear Extra Metadata`

**Comment by RM-FM** [comment]
@wegGehamstert This might be of interest to you.

**Reply by wegGehamstert** [reply]
It worked!
No more `liq_tags` anymore on my files. You are a lifesaver!

**Comment by wegGehamstert** [comment]
I wanted to ask you @RM-FM if you changed something for AutoCue or if you using it just out of the box. Because i like the transitions at your end

**Reply by RM-FM** [reply]
@wegGehamstert That‘s all default… But please be aware that we use sound processing which makes things probably sound even more compact (ducking the old track if the new kicks in).

**Comment by BornKillaz** [comment]
Thank you so much for this. I have recursively added liq_ tags to all files a few days ago... now, it's time to remove them.

It's ok to leave replaygain tags, right? 

Should we also disable Auto Cue in Azuracast after this? I mean, Auto Cue will not work anymore with Liquidsoap 2.3.x, right?

Thank you for your help on this.

**Reply by RM-FM** [reply]
@BornKillaz I would keep them only if you intend to use them instead of autocue calculacted amplify values. Typically, you don't need `replaygain` as an additional feature to autocue. The latest AzuraCast Rolling Release comes with LS 2.3.x and has replaced the `cue_in` implementation of autocue with Liquidsoap's `internal` version. While it functions similarly, there have been reports of issues with long lossless tracks, such as pre-recorded shows.

**Comment by mishcazzulani** [comment]
Hi when I try to run your script i get this error:

ModuleNotFoundError: No module named 'mutagen'

Is installing the Python module enough?
Do I have to run the script inside the VPS hosting Azuracast or should I run it from inside the Docker container?
Thank you!
